source/agora/consensus/protocol/Nominator.d

```diff
@@ -211,9 +212,31 @@ extern(D):
 
     ***************************************************************************/
 
-    public bool receiveEnvelope (SCPEnvelope envelope) @trusted
+    public bool receiveEnvelope (SCPEnvelope envelope) @safe nothrow
```

Comment:
I think this PR needs a rebase. `receiveEnvelope` should no longer return a status code.